cypress/README.md

```diff
@@ -0,0 +1,40 @@
+[![AsyncAPI Banner and Logo](./public/img/logos/github-repobanner-website.png)](https://www.asyncapi.com)
+
+--- 
+## Overview 
+
+This folder contains the test code of AsyncAPI website : 
+
+- It 's powered by [Cypress](https://docs.cypress.io/guides/overview/why-cypress) ,
+
+-It utilises [Cypress/react] to provide some additional functionalites to Cypress . 
+
+## Requirements 
+Use the following tools to set up the project:
```

Comment:
Add the command to do `npm install` with correct npm version and run cypress with appropriate Node version.

Reply:
@reachaadrika This comment is not addressed.